scrc/preprocessors/extractors/spider_specific/bge_reference_extracting_functions.py

```diff
@@ -0,0 +1,44 @@
+from typing import List, Dict, Union
+import bs4
+import re
+
+from typing import Any, Optional
+from scrc.enums.section import Section
+
+
+"""
+This file is used to extract citations from decisions sorted by spiders.
+The name of the functions should be equal to the spider! Otherwise, they won't be invocated!
+Overview of spiders still todo: https://docs.google.com/spreadsheets/d/1FZmeUEW8in4iDxiIgixY4g0_Bbg342w-twqtiIu8eZo/edit#gid=0
+"""
+
```

Comment:
this might be a relic from other extracting functions and probably not relevant here.

Reply:
yes